impl/src/main/java/org/ehcache/config/builders/CacheConfigurationBuilder.java

```diff
@@ -62,9 +65,9 @@
  * instance without modifying the one on which the method was called.
  * This enables the sharing of builder instances without any risk of seeing them modified by code elsewhere.
  */
-public class CacheConfigurationBuilder<K, V> implements Builder<CacheConfiguration<K, V>> {
+public class CacheConfigurationBuilder<K, V> implements FluentCacheConfigurationBuilder<K, V, CacheConfigurationBuilder<K, V>> {
```

Comment:
Should there be some javadoc (package, this class) that discusses that some `ServiceConfiguration` types can be used no more than once while others can be used multiple times?

Reply:
I'm going to document this on `ServiceConfiguration` and `ServiceCreationConfiguration` and the link it from the relevant `FluentConfigurationBuilder` and `FluentCacheConfigurationBuilder` methods.

Reply:
Added the relevant javadoc.